api.go

```diff
@@ -175,6 +184,71 @@ func EquipmentsHandler(manager *DataManager) gin.HandlerFunc {
 	}
 }
 
+func updateParameterTypes(param * FreeFloatingRequestParameter, types []string) {
+	for _, value := range types {
+		enumType := ParseFreeFloatingTypeFromParam(value)
+		if enumType != UnknownType {
+			param.types = append(param.types, enumType)
+		}
+	}
+}
+
+func initParameters(c *gin.Context) (param *FreeFloatingRequestParameter, err error) {
```

Comment:
minor 
```suggestion
func initFreeFloatingRequestParameter(c *gin.Context) (param *FreeFloatingRequestParameter, err error) {
```
Or move every functions related to free floating in it's own separate file.